tests/run_test.sh

```diff
@@ -93,7 +103,15 @@ atparse < ${PATHRT}/fv3_conf/${FV3_RUN:-fv3_run.IN} > fv3_run
 
 atparse < ${PATHRT}/parm/${INPUT_NML:-input.nml.IN} > input.nml
 
-atparse < ${PATHRT}/parm/${MODEL_CONFIGURE:-model_configure.IN} > model_configure
+if [[ $FV3_RUN = 'ccpp_regional_run.IN' ]]; then
```

Comment:
Same here.

Reply:
Exactly. We should not add bunch of if test to add special cases based on various test names or other variables.  `MODEL_CONFIGURE` should be overwritten in each test file.

Reply:
Why all global and regional tests? Only tests that  do not use model_configure.IN, should set, for example:
`export MODEL_CONFIGURE=model_configure_hafs.IN`